vulture/whitelists/unittest.py

```diff
@@ -0,0 +1,7 @@
+from whitelist_utils import Whitelist
+
+test_case = Whitelist()
```

Comment:
test_case -> whitelist_test_case

We have to make sure we don't hide a different variable named `test_case`.